replica/server.go

```diff
@@ -39,6 +39,7 @@ type Server struct {
 	//the replica attempts to connect back to controller
 	MonitorChannel chan struct{}
 	//closeSync      chan struct{}
+	preload bool
```

Comment:
instead of a 'server' variable, can we consider by parameter passing only through all functions? this also becomes a kind of global variable

Reply:
if we are considering to go with this member variable, we need to set this as 'true' by default, and whenever this is unset, it need to be reset to true to assume that this is 'true' by default
However, I prefer passing as argument due to reason mentioned above

Reply:
preload is sent by open() call from controller. If we want to pass it via function arguments again we need to make changes to REST API calls.